repository/concept.go

```diff
@@ -59,10 +60,21 @@ func (r ConceptsRepositoryImp) GetConceptUpdates(ctx context.Context, query bson
 	return updates, nil
 }
 
-func (r ConceptsRepositoryImp) GetConceptUpdate(ctx context.Context, conceptId string, updateId string) (*model.Update, error) {
+func (r ConceptsRepositoryImp) GetConceptUpdate(ctx context.Context, conceptId string, updateId string, page int, size int) (*model.Update, error) {
+	skip := (page - 1) * size
+	limit := size
+
 	filter := bson.D{{Key: "id", Value: updateId}, {Key: "resourceId", Value: conceptId}}
-	bytes, err := r.collection.FindOne(ctx, filter).DecodeBytes()
+	opts := options.Find().SetSkip(int64(skip)).SetLimit(int64(limit))
+
+	//bytes, err := r.collection.FindOne(ctx, filter).DecodeBytes()
```

Comment:
Fjern og flytt opp til GetConceptUpdates
```suggestion
	bytes, err := r.collection.FindOne(ctx, filter).DecodeBytes()
```